hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestStorageContainerManager.java

```diff
@@ -344,7 +346,7 @@ public void testBlockDeletionTransactions() throws Exception {
         } catch (IOException e) {
```

Comment:
@devmadhuu Thanks for working on this.
We are explicitly flushing at L340 `cluster.getStorageContainerManager().getScmHAManager().asSCMHADBTransactionBuffer().flush(),` I think that should flush the transactions, do we still need to define OZONE_SCM_HA_DBTRANSACTIONBUFFER_FLUSH_INTERVAL ?

Reply:
> @devmadhuu Thanks for working on this. We are explicitly flushing at L340 `cluster.getStorageContainerManager().getScmHAManager().asSCMHADBTransactionBuffer().flush(),` I think that should flush the transactions, do we still need to define OZONE_SCM_HA_DBTRANSACTIONBUFFER_FLUSH_INTERVAL ?

Thanks @ashishkumar50 for reviewing the patch. For MiniOzoneCluster, by default SCM HA is not enabled,  so above statement for transaction buffer flush will not execute. So I had to set and reduce the time explicitly for transaction buffer flush.

Reply:
I think this config may be used for HA case only even config name contains "HA flush", It is used for non-HA case also?

Reply:
> I think this config may be used for HA case only even config name contains "HA flush", It is used for non-HA case also?

Yes ideally non HA case, SCM should write deleted block transactions directly to DB, so based on further analysis, we have this method `org.apache.hadoop.hdds.scm.block.DeletedBlockLogStateManagerImpl#addTransactionsToDB`
which adds deleted blocks transactions to DB , and here we are adding to transaction buffer , not directly to DB.
@sumitagrawl do you have an understanding of expected behavior ?

Reply:
explicit call of flush is done in this case, so setting property is redundant

Reply:
I removed that HA flush flag and still can't reproduce the test case failure even after running 300 iterations.
https://github.com/devmadhuu/ozone/actions/runs/6457355734 - > Using Flaky test case workflow
https://github.com/devmadhuu/ozone/actions/runs/6455820324 -> 2 attempts